doc/level1.rst

```diff
@@ -412,9 +412,13 @@ header, by an MUA at receive or display time.
 Messages SHOULD be ignored (i.e., ``peers[from-addr]`` SHOULD NOT be
 updated) in the following cases:
 
+  - The message is explicitly marked as auto-replied with an
+    `Auto-Submitted: auto-replied` header (:rfc:`3834`).
+
```

Comment:
Hum, is it it really right to completely ignore thos?  Delta Chat bots are going to use these headers at some point but still send Autocrypt headers.  Maybe we just need to say that with the Auto-Submitted header we leave the peerstate untouched if there is no Autocrypt header?  If an Autocrypt header is there, it's fine to interpret and update the peer-state i'd think.